src/React/README.md

```diff
@@ -0,0 +1,59 @@
+# Symfony UX React
+
+Symfony UX React integrates [React](https://reactjs.org/) into Symfony applications. It provides tools 
+to render React components from Twig.
+
+**This repository is a READ-ONLY sub-tree split**. See
+https://github.com/symfony/ux to create issues or submit pull requests.
+
+## Usage
+
+```bash
+composer require symfony/ux-react
+```
+
+Then in your `assets/app.js` file, add the following lines:
+
+```
+// assets/app.js
+import { registerReactControllerComponents } from '@symfony/ux-react';
+
+// Registers React controller components to allow loading them from Twig
+//
+// React controller components are components that are meant to be rendered
+// from Twig. These component then rely on other components that won't be called
+// directly from Twig.
+//
+// By putting only controller components in `react/controllers`, you ensure that
+// internal components won't be automatically included in your JS built file if
+// they are not necessary.
+registerReactControllerComponents(require.context('./react/controllers', true, /\.(j|t)sx?$/));
```

Comment:
We MAY soon get the ability to do simple "patches" in Flex. If so, this could be added to a recipe.